actions.py

```diff
@@ -0,0 +1,220 @@
+import logging
+import random
+
+import card as c
+from datetime import datetime
+
+from telegram import Message, Chat
+
+from bot_config import TIME_REMOVAL_AFTER_SKIP
+from errors import DeckEmptyError, NotEnoughPlayersError
+from internationalization import __, _
+from shared_vars import gm, botan
+from user_setting import UserSetting
+from utils import send_async, display_name, game_is_running
+
+logging.basicConfig(
```

Comment:
There's another `logging.basicConfig` in `bot.py`, I think this one should be removed

Reply:
I did it this way because importing logger from bot caused a circular dependency. I'll move logger to shared_vars and use just one then.

Reply:
I wasn't referring to the `logger` though